src/render_engine/themes.py

```diff
@@ -85,7 +85,7 @@ def register_theme(self, theme: Theme):
 
         if theme.template_globals:
             for key, value in theme.template_globals.items():
-                self.engine.globals.setdefault(key, set()).add(value)
+                self.engine.globals.setdefault(key, set()).add(value if not isinstance(value, list) else tuple(value))
```

Comment:
Similar suggestion to the other PR.

```suggestion
                self.engine.globals.setdefault(key, set()).add(value if not isinstance(value, list) else (value))
```